tests/extended/TestAllIndividualPackages.cs

```diff
@@ -93,6 +93,18 @@ async Task TestPackage (string id, string version, string template)
 		var proj_file = Directory.GetFiles (case_dir, "*.csproj").FirstOrDefault ();
 
 		if (proj_file is not null) {
+
```

Comment:
Same as above.

Reply:
There are 2 test projects, so I had to change code in 2 places

Answer is above.

https://github.com/dotnet/android-libraries/pull/1113/files/86d78772d854de8a082b722e4a485a651a8935ad#r2046088731